### PR TITLE
Config param implemented

### PR DESCRIPTION
**Pull request by miros**
--config param was present in code but was not really used, so i implemented it.

### CODE REVIEW

**Comment by savonarola** [comment]
But passing config through a parameter is a vulnerability, since `init-exporter` is intended to be used with suid bit, со this allows for an unprivileged to overwrite system files.

**Comment by andyone** [comment]
@savonarola is right, this is security breach and this is reason why this feature was not implemented.

File: init-exporter.go [hunk 1]
```diff
@@ -165,22 +166,31 @@ func checkArguments() {
 	}
 }
 
+// configPath returns path to config
+func configPath() string {
+	if config_file := arg.GetS(ARG_CONFIG); config_file != "" {
```
Comment (andyone) [inline init-exporter.go]: In this case I recomend to use [`fsutil.ProperPath`](https://godoc.org/pkg.re/essentialkaos/ek.v1/fsutil#ProperPath) method, which return first proper path to config file in slice. 

```go
func configPath() string {
	// Method return first path to readable, non-empty file from given slice
	return fsutil.ProperPath("FRS", []string{arg.GetS(ARG_CONFIG), DEFAULT_CONFIG_FILE})
}
```

**Comment by andyone** [comment]
We can change config priority and value from command line argument will be used only if global config in `/etc` is not present. @savonarola what do you think?

File: init-exporter.go [hunk 2]
```diff
@@ -67,13 +67,14 @@ const (
 	FORMAT_SYSTEMD = "systemd"
 )
 
-// CONFIG_FILE contains path to config file
-const CONFIG_FILE = "/etc/init-exporter.conf"
+// DEFAULT_CONFIG_FILE contains path to config file
+const DEFAULT_CONFIG_FILE = "/etc/init-exporter.conf"
```
Comment (andyone) [inline init-exporter.go]: Using this constant is wrong way. You can just set default value for config argument:
```go
ARG_CONFIG:    &arg.V{Value:"/etc/init-exporter.conf"},
```
In this case, if `--config` is not present, `arg.GetS(ARG_CONFIG)` return path to default value.

**Comment by savonarola** [comment]
@andyone  I think that using config from command line argument when global config is missing is also dangerous, because global config can be absent due to some misconfig.

Maybe we should add an explicit parameter to global config witch allows to use `--config` command line option?

**Comment by andyone** [comment]
I not sure. I think @miros implement this feature for testing purposes and using global config property which allows define user config doesn't solve this problem.

**Pull request closed without merging**